src/zjs_ocf_ble.c

```diff
@@ -44,21 +33,11 @@ static int str2bt_addr_le(const char *str, const char *type, bt_addr_le_t *addr)
         return -EINVAL;
     }
 
-    for (i = 5, j = 1; *str != '\0'; str++, j++) {
-        if (!(j % 3) && (*str != ':')) {
+    for (i = 5; *str != '\0', i >= 0; str+=3, i--) {
```

Comment:
I think this comma will only check the value after the comma, you really mean &&